jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -196,33 +199,34 @@ public boolean clearCurrentSelection( CellPreviewEvent<ProcessInstanceSummary> e
 
                     @Override
                     public DefaultSelectionEventManager.SelectAction translateSelectionEvent( CellPreviewEvent<ProcessInstanceSummary> event ) {
+                        DefaultSelectionEventManager.SelectAction ret = DefaultSelectionEventManager.SelectAction.DEFAULT;
                         NativeEvent nativeEvent = event.getNativeEvent();
                         if ( BrowserEvents.CLICK.equals( nativeEvent.getType() ) ) {
                             // Ignore if the event didn't occur in the correct column.
-                            if ( extendedPagedTable.getColumnIndex( actionsColumn ) == event.getColumn() ) {
```

Comment:
@rbarriuso actually, after reviewing a bit more of how you fixed it, it is a bit particular to the process instances selection because you also have to ignore the error count column. It is still valid that we create a new JIRA to handle the other views, but for this one your changes are good to me.